pkg/collector/api.go

```diff
@@ -89,7 +88,6 @@ func (s *APIServer) storeRecordHandler() httprouter.Handle {
 			writeError(w, http.StatusBadRequest, fmt.Errorf("failed to decode record: %v", err))
 			return
 		}
-		rec.Timestamp = strconv.FormatInt(time.Now().Unix(), 10)
```

Comment:
I did this on purpose, to add some modicum of sanity to the data, but I guess it doesn't matter much.